timeline-for-beaver-builder.php

```diff
@@ -27,15 +27,15 @@ class BSFBBTimeline {
 		/**
 		 * To initializ new object
 		 */
-		function __construct() {
+		public function __construct() {
```

Comment:
For above errors add above class name related errors in phpcs ignore list

Reply:
I have ignored it in ruleset for some, but somehow it's failing for other checks